### PR TITLE
Add branding for cliqz and alpha packages

### PR DESCRIPTION
**Pull request by joseph-cliqz**
@spacifici

### CODE REVIEW

**Comment by alver-cliqz** [comment]
Is it any ruby was in Fennec project before? If not - I suggest not to add new language dependencies.

**Comment by joseph-cliqz** [comment]
@alver-cliqz I have removed the Gemfiles but in another PR I will be adding fastlane to the project which requires ruby

File: mozilla-release/mobile/android/branding/cliqz-alpha/configure.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+ANDROID_PACKAGE_NAME=com.cliqz.browser.alpha
+MOZ_APP_DISPLAYNAME="Cliqz Browser"
```
[the remark this answers is not in the document]
Comment (joseph-cliqz) [reply mozilla-release/mobile/android/branding/cliqz-alpha/configure.sh]: there is a folder for cliqz as well.

File: mozilla-release/mobile/android/branding/cliqz-alpha/locales/en-US/brand.dtd [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!ENTITY  brandShortName  "Fennec">
```
[the remark this answers is not in the document]
Comment (spacifici) [reply mozilla-release/mobile/android/branding/cliqz-alpha/locales/en-US/brand.dtd]: There is a separate task for that. This is a simple skeleton that we need for Fastlane

**Pull request merged**